app/controllers/apps_controller.rb

```diff
@@ -29,7 +29,17 @@ def index
         raise ActionController::RoutingError.new('Not Found') unless app.accessible?
       end
 
-      @motd_file = MotdFile.new
+      motd_file = MotdFile.new(ENV['MOTD_PATH'])
+      case ENV['MOTD_FORMAT']
+        when 'osc'
+          @motd = MotdFormatterOsc.new(motd_file)
+        when 'markdown'
+          @motd = MotdFormatterMarkdown.new(motd_file)
+        when 'rss'
+          @motd = MotdFormatterRss.new(motd_file)
+        else
+          @motd = MotdFormatterPlaintext.new(motd_file)
+      end
```

Comment:
Added `end if motd_file.exist?`, which ensures the `@motd` is not set if the `ENV['MOTD_PATH']` is `nil` or invalid.